src/pseudopeople/interface.py

```diff
@@ -14,15 +16,28 @@
 from pseudopeople.schema_entities import COLUMNS, DATASETS, Dataset
 from pseudopeople.utilities import configure_logging_to_terminal, get_state_abbreviation
 
+if TYPE_CHECKING:
+    import modin.pandas
+
+    # TODO: Using type aliases isn't very good for autodoc purposes,
+    # since it displays the alias and not its actual value.
+    # When we have more engines and these types are more complicated,
+    # we might consider exposing them as a public module on pseudopeople,
+    # then referencing them.
+    # But for now, we just duplicate the full types everywhere so there
+    # is less indirection in the docs.
+    # from pseudopeople.utilities import DATAFRAME, ENGINE
```

Comment:
Autodocs concerns are out of scope for a spike on modin, so I recommend removing all your changes related to this. We should have a separate conversation about how we want to manage that.